.vsts.pipelines/jobs/ci-linux_bootstrap.yml

```diff
@@ -0,0 +1,106 @@
+parameters:
+  job: null
+  matrix:
+    Production: {}
+  pool:
+    name: Hosted Ubuntu 1604
+  imageName: null
+  reportPrebuiltLeaks: false
+  dependsOn: ''
+  generatePrebuiltBurndown: false
+  runUnitTests: false
```

Comment:
Do you need the reportPrebuiltLeaks, generatePrebuiltBurndown or runUnitTests parameters?  It doesn't appear that they're used.